index.html

```diff
@@ -204,6 +207,7 @@ <h4 class="lets-connect">LET'S CONNECT</h4>
                 <p>Skills</p><i class="right arrow"></i>
             </li>
         </ul>
+        </div>
     </section>
     <footer class="contact-page">
        <h2 class="contactme">Contact me</h2>
```

Comment:
- [ ] Please take some steps back to the mobile version and fix this broken style, yes.. we need to have the mobile version intact, not broken because of modification brought to the desktop, check this out, there is a space at the bottom and on the left of your page.
- [ ] The background image is hiding a piece of text, what do you think if you adjust it a bit ? I think it is important for that text to be visible.

<img width="375" alt="yes" src="https://user-images.githubusercontent.com/77843050/206583068-72449f75-2de6-473c-9d0d-4d89047c0157.png">